### PR TITLE
Uses Kotlin tuples

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This has better typing.

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 1]
```diff
@@ -28,7 +28,10 @@ import android.content.res.Resources
 import android.graphics.Color
 import android.media.MediaPlayer
 import android.net.Uri
-import android.os.*
+import android.os.Build
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:31]: revert all unnecessary import changes

File: AnkiDroid/src/main/java/com/ichi2/utils/Triple.kt [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
- * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A      *
- * PARTICULAR PURPOSE. See the GNU General Public License for more details.             *
- *                                                                                      *
- * You should have received a copy of the GNU General Public License along with         *
- * this program.  If not, see <http://www.gnu.org/licenses/>.                           *
- ****************************************************************************************/
-
-package com.ichi2.utils
-
-class Triple<First, Second, Triple> constructor(val first: First, val second: Second, val third: Triple)
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/utils/Triple.kt:19]: Very surprised this compiled (it does!)

**Review by david-allison — CHANGES_REQUESTED** [review]
In general, looks good. Thanks!

Diffs need to be cleaned up via fixing the imports which were unnecessarily changed)

File: AnkiDroid/src/main/java/com/ichi2/utils/Triple.kt (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/utils/Triple.kt:19]: why surprised?

Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/utils/Triple.kt:19]: References weren't changed in the other files in this commit

**Review by david-allison — APPROVED** [review]

**Pull request merged**